pandas/io/stata.py

```diff
@@ -1261,7 +1265,7 @@ def _get_fmtlist(self):
 
     # Returns the label list
     def _get_lbllist(self):
-        if self.format_version >= 118:
+        if self.format_version in (118, 119):
```

Comment:
Was the `>=` before a mistake or is there a reason why this needed to change?

Reply:
I reverted these to all be >=.